### DISCUSSION TITLE
BTT_TFT Thumbnail works only if HEADER_BLOCK is manually removed

### DISCUSSION THREAD

**Discussion by kamilkrz** [OrcaSlicer/OrcaSlicer#7703] [Q&A] [answered]
Hi,
I just flashed my Artillery genius pro to combo of Marlin 2.1.2.1 and MKS_GD_TFT28_V4.0
Now I am trying to get thumbnails to work.
I was able to find that 70x70/BTT_TFT is correct format but unfortunately those only works if I manually remove whole HEADER_BLOCK section.

It simply looks like thumbnail has to be before that block.
Is there any advice how to achieve this on Orca?

TIA

**Comment by Marcos2BR** [comment, marked as answer]
I think they corrected this in version 2.3.0b

see https://github.com/SoftFever/OrcaSlicer/pull/7488

**Reply by kamilkrz** [reply]
Thanks a lot ;)

**Reply by Marcos2BR** [reply]
Friend, were you able to see the thumbnails in TFT with version V2.3.0-dev?
I can only do it if I edit the gcode in notepad++, I sent a message to the "staff" there in the link

**Reply by kamilkrz** [reply]
Yup - there is some problem with added gcode for labeling objects (my marlin build has apparently not this feature enabled) but otherwise - pretty thumbnails

**Reply by kamilkrz** [reply]
@Marcos2BR I am using linux build
